tests/group_functions/group_scan.h

```diff
@@ -337,35 +338,76 @@ struct ScanOverGroupDataStruct {
     T init_value = with_init ? T(init) : sycl::known_identity<OpT, T>::value;
     // res consists of 4 series of results: two pairs of exclusive and inclusive
     // scan results made over 'group' and 'sub_group' accordingly.
-    for (int group_i = 0; group_i < 2; group_i++) {
-      std::string group_name = group_i == 0 ? "group" : "sub_group";
-      size_t group_offset = range_size * group_i;
+    {
+      std::string group_name = "group";
```

Comment:
We might as well inline these again now that they don't share the loop.

Reply:
Done.